src/components/__tests__/LinnWeekPicker.test.js

```diff
@@ -16,7 +16,7 @@ describe('<LinnWeekPicker />', () => {
                 selectedDate={moment('11-09-2019', 'MM-DD-YYYY')}
             />
         );
-        const item = getByDisplayValue('09/11/2019');
+        const item = getByDisplayValue('2019-11-09T00:00:00+00:00');
```

Comment:
Should these still not be displaying the formatted values?

Reply:
I thought this was funky too... but I checked it in production and it looks fine.